app/Actions/CompileAssets.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Actions;
+
+use App\Shell\Shell;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\File;
+
+class CompileAssets
+{
+    use LamboAction;
+
+    protected $shell;
+
+    public function __construct(Shell $shell)
+    {
+        $this->shell = $shell;
+    }
+
+    public function __invoke()
+    {
+        if (config('lambo.store.mix') || config('lambo.store.full')) {
+
+            $this->squashProgress();
+
+            $this->logStep('Compiling project assets');
+
+            $process = $this->shell->execInProject("npm run dev-no-progress {$this->extraOptions()}");
+
+            $this->abortIf(! $process->isSuccessful(), 'Compilation of project assets did not complete successfully', $process);
+
+            $this->info('Project assets compiled successfully.');
+        }
```

Comment:
After running, please move the backup over back in as `package.json`

Reply:
Done.